lib/engine/game/g_1848/game.rb

```diff
@@ -588,7 +592,7 @@ def operating_order
         end
 
         def upgrades_to?(from, to, _special = false, selected_company: nil)
-          return %w[5 6 57].include?(to.name) if (from.hex.tile.label.to_s == 'K') && (from.hex.tile.color == 'white')
+          return %w[5 6 57].include?(to.name) if (from.hex.tile.label.to_s == 'K') && (from.hex.tile.color == :white)
```

Comment:
@ventusignis has a whole scheme for handling label upgrades. Talk to him about how he did it for other games.